### ISSUE TITLE
Support keyboard interactive login

### ISSUE THREAD

**Issue by isontheline** [isontheline/pro.webssh.net#140]
From the App Store :

# Very questionable security 
by Motobear1400 – 15 déc. 2020
Tell me developer(s)...

Why is it that you forbid the most standard of all types of ssh login, where you don’t have to input sensitive data into an app ‘box’ but rather just be connected to the remote host and enter that data in real time as it’s requested? This is by far the most standard configuration, and what you’re doing spells scam or danger, and introduces more red flags than a 60+ year old dude dressed in woman’s clothing and selling services in a red light district...

What gives?

**Comment by isontheline** [comment]
Hi Motobear1400 👋

I like your spicy review 🌶   thank you!

So what gives?

1. Tell me developer(s) : Only me 😇
2. "Why is it that you forbid the most standard of all types of ssh login[...]" : You're right Moto, but why don't you have opened an issue? I'm so happy with your review that I have created it for you : 
Kind isn't it?
3. "[...]old dude dressed in woman’s clothing and selling services in a red light district[...]" : Hum Hum, stay calm Moto I will add interactive keyboard login a soon as possible.

So I gave you.

And, have a nice day Moto ☀️
